docs/cody/clients/enable-cody-enterprise.mdx

```diff
@@ -1,101 +1,119 @@
-# Cody on Sourcegraph Enterprise
+# Cody for Enterprise
 
 <p className="subtitle">Cody enhances your coding experience by providing intelligent code suggestions, context-aware completions, and advanced code analysis. These docs will help you use Cody on your Sourcegraph Enterprise instance.</p>
 
 <LinkCards>
    <LinkCard href="https://sourcegraph.com/contact/request-info" imgSrc="https://sourcegraph.com/.assets/img/sourcegraph-mark.svg" imgAlt="Cody Enterprise" title="Cody Enterprise" description="Get in touch with our team to try Cody for Sourcegraph Enterprise." />
 </LinkCards>
 
-## Cody Enterprise features
+## Setting up Cody Enterprise
 
-To cater to your Enterprise requirements, Cody offers the following features:
+You can set up Cody for your Enterprise instance by two methods:
 
-### IDE token expiry
+1. Sourcegraph Cloud
+2. Self-hosted Sourcegraph
 
-Site administrators can set the duration of access tokens for users connecting Cody from their IDEs (VS Code, JetBrains, etc.). This can be configured from the **Site admin** page of the Sourcegraph Enterprise instance. Available options include **7, 14, 30, 60, and 90 days**.
+## Cody on Sourcegraph Cloud
 
-![ide-token-expiry](https://storage.googleapis.com/sourcegraph-assets/Docs/token-expiry.png)
+With [Sourcegraph Cloud](/cloud/), you get Cody as a managed service, and you **do not** need to enable Cody as is required for self-hosted setup. However, by contacting your account manager, Cody can still be enabled or disabled on-demand on your Sourcegraph instance.
 
-### Guardrails
+## Self-hosted Sourcegraph Enterprise
 
-<Callout type= "note">Guardrails for public code is currently in Beta and is supported with VS Code and JetBrains IDEs extensions.</Callout>
+### Prerequisites
 
-Open source attribution guardrails for public code, commonly called copyright guardrails, reduce the exposure to copyrighted code. This involves implementing a verification mechanism within Cody to ensure that any code generated by the platform does not replicate open source code.
+- You have Sourcegraph version `5.1.0` or more
+- A Sourcegraph Enterprise subscription with [Cody Gateway](/cody/core-concepts/cody-gateway) or an account with a third-party LLM provider
 
-Guardrails for public code are available to all Sourcegraph Enterprise instances and are **disabled** by default. You can enable them from the Site configuration section by setting `attribution.enabled` to `true`.
+### Enable Cody on your Sourcegraph instance
 
-Guardrails don't differentiate between license types. It matches any code snippet that is at least **ten lines** long from the **290,000** indexed open source repositories.
+Site admins can only enable Cody on the Sourcegraph instance. To do so,
 
-### Admin controls
+- First, configure your desired LLM provider either by [using Sourcegraph Cody Gateway](/cody/core-concepts/cody-gateway) (recommended) or by directly using a third-party LLM provider
```

Comment:
Nit: from a users POV, reading "recommended" raises the question: why is Cody Gateway the recommended setup?
If I wanted to make this doc outstanding, I'd clarify this "why" here.

Reply:
https://github.com/sourcegraph/docs/pull/735/files?diff=split&w=1#diff-42526137bd57614d87089e19ecc664312c43ac87930336bbe3d9aeee9bba666aR11 is a good explanation. Maybe reference that section here, like "recommended, [see why](#Using Sourcegraph Cody Gateway)"